cross/acl/PLIST

```diff
@@ -0,0 +1,7 @@
+bin:bin/chacl
+bin:bin/getfacl
+bin:bin/setfacl
+lnk::libexec/libacl.so
```

Comment:
seems like there is a `:` to much

Reply:
I do not know about this: I used spksrc/cross/attr/PLIST as a reference for building this package and it HAS double ::

Reply:
well it's wrong there then too. AFAIK there should always only be one reference: https://github.com/SynoCommunity/spksrc/wiki/PLIST-files#example